doc/deploy_documentation.sh

```diff
@@ -23,14 +23,10 @@ if [ -d "$dir/html" ] && [ -s "$dir/html/index.html" ]; then
     # Add everything in this directory (the Doxygen code documentation)
     git add -A .
     # Commit the added files with a title and description containing the Travis CI
-    git commit -m "Deploy code docs to GitHub Pages.
-
-
-     Travis build info:
-     branch name ${TRAVIS_BRANCH}
-     build number${TRAVIS_BUILD_NUMBER}"
+    git commit -m "Deploy docs to GitHub Pages.Travis build info: \
+     branch name ${TRAVIS_BRANCH} build number${TRAVIS_BUILD_NUMBER}"
```

Comment:
can you leave a blank line after `Deploy docs to GitHub Pages` ? (also, remove `.`)